packages/admin/cms-admin/src/userPermissions/user/permissions/ContentScopeGrid.tsx

```diff
@@ -59,51 +47,85 @@ export const ContentScopeGrid = ({ userId }: { userId: string }) => {
         return <Loading />;
     }
 
+    const columns: GridColDef<ContentScope>[] = generateGridColumnsFromContentScopeProperties(data.userContentScopes);
+
     return (
-        <Card>
-            <FinalForm<FormValues>
-                mode="edit"
-                onSubmit={submit}
-                onAfterSubmit={() => null}
-                initialValues={{ contentScopes: data.userContentScopes.map((cs) => JSON.stringify(cs)) }}
-            >
+        <>
+            <Card>
                 <CardToolbar>
                     <ToolbarTitleItem>
                         <FormattedMessage id="comet.userPermissions.scopes" defaultMessage="Scopes" />
                     </ToolbarTitleItem>
                     <ToolbarFillSpace />
                     <ToolbarActions>
-                        <SaveButton type="submit">
-                            <FormattedMessage id="comet.userPermissions.save" defaultMessage="Save" />
-                        </SaveButton>
+                        <Button startIcon={<Select />} onClick={() => setOpen(true)} variant="contained" color="primary">
+                            <FormattedMessage id="comet.userPermissions.selectScopes" defaultMessage="Select scopes" />
+                        </Button>
                     </ToolbarActions>
                 </CardToolbar>
                 <CardContent>
-                    {data.availableContentScopes.map((contentScope: ContentScope) => (
-                        <Field
-                            disabled={data.userContentScopesSkipManual.some((cs: ContentScope) => isEqual(cs, contentScope))}
-                            key={JSON.stringify(contentScope)}
-                            name="contentScopes"
-                            fullWidth
-                            variant="horizontal"
-                            type="checkbox"
-                            component={FinalFormCheckbox}
-                            value={JSON.stringify(contentScope)}
-                            label={Object.entries(contentScope).map(([scope, value]) => (
-                                <>
-                                    {camelCaseToHumanReadable(scope)}: {camelCaseToHumanReadable(value)}
-                                    <br />
-                                </>
-                            ))}
-                        />
-                    ))}
+                    <DataGrid
+                        autoHeight={true}
+                        rows={data.userContentScopes ?? []}
+                        columns={columns}
+                        rowCount={data?.userContentScopes.length ?? 0}
+                        loading={false}
+                        getRowHeight={() => "auto"}
+                        getRowId={(row) => JSON.stringify(row)}
+                        sx={{ "&.MuiDataGrid-root .MuiDataGrid-cell": { py: "8px" } }}
```

Comment:
Why is this necessary?

Reply:
What happens if we remove them? AFAIK it shouldn't be necessary to have custom styling in the Admin.

Reply:
i removed it, you were right it does not change anything :) probably some leftover from the past :)